site/src/pug/includes/body.pug

```diff
@@ -25,7 +25,7 @@ include ./fork.pug
                     .select-wrapper
                         label.sr-only(for="sorting") Sort opportunities
                         select#sorting
-                            for filter in ['Alphabetical', 'Alphabetical, reversed', 'Difficulty', 'Difficulty, reversed']
+                            for filter in ['Alphabetical', 'Alphabetical, reversed', 'Difficulty', 'Difficulty, reversed', 'Popular', 'Popular, reversed']
```

Comment:
This should be programmatically applied after the claps come in from the API - otherwise the logic for sort-by-popularity won't work

i.e.
```js
requestClaps(urls).then(resp => {
	claps = response;
	const rNode = document.createElement('option');
	node.value = 'popularity';
	node.textContent = 'Popularity';
	// ... create the node for reverse
	selectInput.appendChild(node);
	selectInput.appendChild(rNode);
});